### PR TITLE
Fixed typos, tidied up, and applied style guide

### PR DESCRIPTION
**Pull request by catherinepope**
# Description
Fixed typos, tidied up, and applied style guide

# Reasons
https://circleci.atlassian.net/browse/DOCSTEAM-101

### CODE REVIEW

File: jekyll/_cci2/server-3-overview.adoc [hunk 1]
```diff
@@ -245,37 +244,37 @@ but can also be configured to be exported to other places (i.e. Datadog or Prome
...
 | Licensing
-| Deploys and controls Kotsadm
+| Deploys and controls Kotsadm.
 |
 |
 
 | kotsadm-postgres
 | Licensing
-| Database for Kots licensing
+| Database for Kots licensing.
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-overview.adoc:265]:
```suggestion
| Database for KOTS licensing.
```
```suggestion
| Database for Kots licensing.
```

File: jekyll/_cci2/server-3-whats-new.adoc [hunk 2]
```diff
@@ -158,31 +158,30 @@ Remove the older Traefik pod with the following command:
 kubectl delete pod circleci-server-traefik-<older pod hash>
 ```
 
-The new Traefik pos will then start ot schedule correctly.
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-whats-new.adoc:161]: 😆

File: jekyll/_cci2/server-3-whats-new.adoc [hunk 3]
```diff
@@ -259,7 +251,7 @@ and get details of the possible cause of the issue.
 
 == Release 3.0.1
 
-- `build_agent` `version` value updated, as the previous version was relying on a vulnerable version of PsExec.
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-whats-new.adoc:262]: I am not sure here, let's leave as is for now as this item will have been checked by engineering.

Comment (catherinepope) [reply jekyll/_cci2/server-3-whats-new.adoc]: OK, have reverted to previous version.

**Pull request merged**